packages/ui/src/components/Designer/Main/index.tsx

```diff
@@ -240,20 +232,7 @@ const Main = (props: Props, ref: Ref<HTMLDivElement>) => {
   };
 
   return (
-    <div
-      ref={ref}
-      onClick={(e) => {
-        e.stopPropagation();
```

Comment:
I'm not sure if it's still necessary to keep a `stopPropagation()` here?

Reply:
I can't remember...
I was able to confirm that it works correctly with the image type schema, so I decided that removing it was not a problem.